libcxx/test/std/language.support/support.exception/propagation/make_exception_ptr.pass.cpp

```diff
@@ -39,7 +39,7 @@ int main(int, char**)
         }
         catch (const A& a)
         {
-#ifndef _LIBCPP_ABI_MICROSOFT
+#ifndef TEST_ABI_MICROSOFT
```

Comment:
`libcxx/test/libcxx/memory/trivial_abi/unique_ptr_destruction_order.pass.cpp` is probably not tested by MSVC STL but can you update that file too?